README.md

```diff
@@ -137,6 +137,8 @@ Attestations are saved in the JSON-serialized [Sigstore bundle][6] format.
 
 If multiple subjects are being attested at the same time, a single attestation
 will be created with references to each of the supplied subjects.
+If `single-subject-attestations` is true, each attestation will be written
+to the output file on  a separate line (using the [JSON Lines][7] format).
```

Comment:
Technically, JSON Lines is used regardless, but in the multi-subject case there is only ever one line